lib/fauna/query.rb

```diff
@@ -1,4 +1,7 @@
 module Fauna
+
+  extend self
```

Comment:
So you can call `Fauna.query`, and if you include Fauna, unscoped `query` also works.